tests/e2e/specs/trade-spec.js

```diff
@@ -3,33 +3,37 @@ import ExchangePage from '../pages/exchange/exchange-page';
 const exchange = new ExchangePage();
 const testedAsset = 'sETH';
 
-describe('Trades tests', () => {
+let metamaskWalletAddress;
+
+/*
+* This end-to-end test illustrates the happy flow of swapping synths through the Kwenta exchange on Optimistic Kovan
+* 1 sUSD is swapped
+* This e2e test works with Chrome on Optimistic Kovan   
+*/
+
+describe('Trade 1 sUSD for sETH on Optimism', () => {
 	context(`Trade sUSD => ${testedAsset}`, () => {
 		before(() => {
-			exchange.snxExchangerSettle(testedAsset).then((settleTx) => {
-				if (settleTx) {
-					exchange.waitUntilAvailableOnEtherscan(settleTx, 'settleTx');
-				}
+			exchange.getMetamaskWalletAddress().then((address) => {
+				metamaskWalletAddress = address;
```

Comment:
What is `metamaskWalletAddress` used for?

Reply:
I actually thought this was a necessity to get the test fixed (basically calling a synpress / cypress function)  . However I did some additional digging and I think I can clean this up further. So that's good: less code with the same result.